science-platform/templates/alert-stream-broker-application.yaml

```diff
@@ -24,6 +24,14 @@ spec:
     repoURL: {{ .Values.repoURL }}
     targetRevision: {{ .Values.revision }}
     helm:
+      parameters:
+        - name: "global.host"
+          value: {{ .Values.fqdn | quote }}
+        - name: "global.baseUrl"
+          value: "https://{{ .Values.fqdn }}"
+        - name: "global.vaultSecretsPath"
```

Comment:
I think I called this `global.vaultSecretsPathPrefix` in my apps; I don't know if you want to do the same?

Reply:
I'd be good with that, but we should be consistent one way or the other across all applications, so we should probably fix the existing ones in one direction or the other.

Reply:
Y'all figure out which way it should go, and I'm happy to make it consistent.  I prefer the shorter version, myself.